server/src/main/java/dev/slimevr/platform/SteamVRBridge.java

```diff
@@ -153,117 +154,159 @@ protected void writeBatteryUpdate(Tracker localTracker) {
 		List<Tracker> allTrackers = Main.getVrServer().getAllTrackers();
 		TrackerRole role = localTracker.getTrackerPosition().getTrackerRole();
 
-		Tracker primaryTracker = null;
-		Tracker secondaryTracker = null;
-		Tracker tertiaryTracker = null;
-		Tracker fourthTracker = null;
+		List<Tracker> batteryTrackers = new FastList<>();
 
 		// Given what the role is of localTracker, the tracker positions that
 		// make up that role are set to primaryTracker, secondaryTracker, and
 		// tertiaryTracker respectively.
-		primaryTracker = TrackerUtils
-			.getNonInternalTrackerForBodyPosition(
-				allTrackers,
-				TrackerPosition.getByTrackerRole(role)
+		batteryTrackers
+			.add(
+				TrackerUtils
+					.getNonInternalTrackerForBodyPosition(
+						allTrackers,
+						TrackerPosition.getByTrackerRole(role)
+					)
 			);
 		switch (role) {
 			case WAIST -> {
-				secondaryTracker = TrackerUtils
-					.getNonInternalTrackerForBodyPosition(
-						allTrackers,
-						TrackerPosition.WAIST
+				batteryTrackers
+					.add(
+						TrackerUtils
+							.getNonInternalTrackerForBodyPosition(
+								allTrackers,
+								TrackerPosition.WAIST
+							)
 					);
 				// When the chest SteamVR tracking point is disabled, aggregate
 				// its battery level alongside waist and hip.
 				if (!(config.getBridgeTrackerRole(TrackerRole.CHEST, true))) {
-					tertiaryTracker = TrackerUtils
-						.getNonInternalTrackerForBodyPosition(
-							allTrackers,
-							TrackerPosition.CHEST
+					batteryTrackers
+						.add(
+							TrackerUtils
+								.getNonInternalTrackerForBodyPosition(
+									allTrackers,
+									TrackerPosition.CHEST
+								)
 						);
-					fourthTracker = TrackerUtils
-						.getNonInternalTrackerForBodyPosition(
-							allTrackers,
-							TrackerPosition.UPPER_CHEST
+					batteryTrackers
+						.add(
+							TrackerUtils
+								.getNonInternalTrackerForBodyPosition(
+									allTrackers,
+									TrackerPosition.UPPER_CHEST
+								)
```

Comment:
This is more of a stylistic comment, but I think it's better to order tracker positions based on their precedence. So, if upper chest is the more common position over chest, then upper chest should come first. Since it's part of a list, it doesn't really matter at any rate.

Reply:
They are ordered by their precedence, upper chest should have less priority than chest. But let's see Erimel's response

Reply:
I think it should go from the nearest for the furthest, so if we ever want to do some weighted average or something based on "which tracker is closest to the tracking point", we can. Just pushed a commit for that

Reply:
> They are ordered by their precedence, upper chest should have less priority than chest.

Not with the chest, though. This was under waist. If upper chest is the primary tracker position for the chest role, then chest is the secondary tracker position for the chest role.